tensorflow/core/api_def/base_api/api_def_AsString.pbtxt

```diff
@@ -42,5 +42,11 @@ Supports many numeric types and boolean.
 For Unicode, see the
 [https://www.tensorflow.org/tutorials/representation/unicode](Working with Unicode text)
 tutorial.
+
+Examples:
+>>> tf.strings.as_string([3, 2])
```

Comment:
There needs to be a newline between `Examples:` and the doctest. Please do so in all files.

Reply:
Fixed in `dad8c5d`